mmdet/models/necks/fpn.py

```diff
@@ -22,10 +22,18 @@ class FPN(nn.Module):
             build the feature pyramid. Default: 0.
         end_level (int): Index of the end input backbone level (exclusive) to
             build the feature pyramid. Default: -1, which means the last level.
-        add_extra_convs (bool): Whether to add conv layers on top of the
-            original feature maps. Default: False.
-        extra_convs_on_inputs (bool): Whether to apply extra conv on
-            the original feature from the backbone. Default: False.
+        add_extra_convs (bool | str): If bool, it decides whether to add conv
+            layers on top of the original feature maps. Default to False.
+            If True, its actual mode is specified by `extra_convs_on_inputs`.
+            If str, it specifies the source feature map of the extra convs.
+            Only the following options are allowed
+             'on_input': Last feat map of neck inputs (i.e. backbone feature).
+             'on_lateral':  Last feature map after lateral convs.
+             'on_output': The last output feature map after fpn convs.
```

Comment:
The format will not be correctly parsed in the doc.
```suggestion
            Only the following options are allowed
            
            - 'on_input': Last feat map of neck inputs (i.e. backbone feature).
            - 'on_lateral':  Last feature map after lateral convs.
            - 'on_output': The last output feature map after fpn convs.
```